### PR TITLE
Update Running-Alluxio-Locally.md

### PR DESCRIPTION
**Pull request by armkhachatryan**
Added a section for mounting RAMFS file system before formatting it to avoid errors like - "java.nio.file.NoSuchFileException: /mnt/ramdisk"

### CODE REVIEW

File: docs/en/deploy/Running-Alluxio-Locally.md [hunk 1]
```diff
@@ -35,6 +35,13 @@ repeatedly input the password, you can add the public SSH key for the host into
 `~/.ssh/authorized_keys`. See [this tutorial](http://www.linuxproblem.org/art_9.html) for more
 details.
 
+## Mount RAMFS file system
+> Run the belo command to mount RAMFS file system.
```
Comment (ZacBlanco) [inline docs/en/deploy/Running-Alluxio-Locally.md]: Typo on `below`?

Comment (ZacBlanco) [reply docs/en/deploy/Running-Alluxio-Locally.md]: Also, can you put a line break after the section title?

Comment (armkhachatryan) [reply docs/en/deploy/Running-Alluxio-Locally.md]: Done

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Comment by apc999** [comment]
alluxio-bot, cherry-pick this to branch-2.0 please

**Pull request merged**